interface-definitions/include/accel-ppp/default-pool.xml.i

```diff
@@ -0,0 +1,14 @@
+<!-- include start from accel-ppp/default-pool.xml.i -->
+<leafNode name="default-pool">
+  <properties>
+    <help>Default client ip pool name</help>
```

Comment:
"IP" is an abbreviation and should be all-caps.

Reply:
done